<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefDialog</class>
 <widget class="QDialog" name="PrefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="renderingBox">
     <property name="title">
      <string>Rendering algorithm</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="interpolationCheckBox">
        <property name="text">
         <string>Linear interpolation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mipCheckBox">
        <property name="text">
         <string>Maximum intensity projection (MIP)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qualityBox">
     <property name="title">
      <string>Rendering quality</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="movingLayout">
        <item>
         <widget class="QLabel" name="movingLabel">
          <property name="text">
           <string>Moving</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="movingDial">
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="movingSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="stillLayout">
        <item>
         <widget class="QLabel" name="stillLabel">
          <property name="text">
           <string>Still</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="stillDial">
          <property name="wrapping">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="stillSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>PrefDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
